src/app/modules/home/components/events-carousel/events-carousel.component.html

```diff
@@ -0,0 +1,20 @@
+@if (events.length == 0) {
+  <p class="text-xl font-semibold place-self-center">No Events Currently Available</p>
+}
+
+@if (events.length > 0) {
```

Comment:
could use else if, but also the first if statement you can make it `<= 0` since you only cover cases of `== 0` or `> 0`